### PR TITLE
Feature/avo 1875

### PR DESCRIPTION
**Pull request by ian-emsens-shd**
https://meemoo.atlassian.net/browse/AVO-1875

### CODE REVIEW

File: src/collection/components/CollectionOrBundleOverview.tsx [hunk 1]
```diff
@@ -364,14 +391,22 @@ const CollectionOrBundleOverview: FunctionComponent<CollectionOrBundleOverviewPr
...
 					break;
+
+				case 'createQuickLane':
+					setSelected(collectionId);
+					setIsQuickLaneModalOpen(true);
+					break;
+
 				case 'delete':
 					onClickDelete(collectionId);
 					break;
```
Comment (ian-emsens-shd) [inline src/collection/components/CollectionOrBundleOverview.tsx:408]: Dit is dus het probleem met `onClick(whichButton: string) => void`-type componenten

Comment (bertyhell) [reply src/collection/components/CollectionOrBundleOverview.tsx:408]: welk probleem bedoel je precies?

File: src/shared/translations/nl.json [hunk 2]
```diff
@@ -1799,6 +1799,7 @@
   "collection/views/collection-overview___bent-u-zeker-deze-actie-kan-niet-worden-ongedaan-gemaakt": "Ben je zeker dat je deze collectie wil verwijderen?<br />Deze actie kan niet ongedaan gemaakt worden.",
   "collection/views/collection-overview___beschrijving-hoe-collecties-aan-te-maken": "Een collectie is een verzameling van video- of audiofragmenten rond een bepaald thema of voor een bepaalde les. Nadat je een collectie hebt aangemaakt kan je deze publiek maken voor andere gebruikers op het platform.",
   "collection/views/collection-overview___bewerk": "Bewerk",
+  "collection/views/collection-overview___delen-met-leerlingen": "",
```
Comment (bertyhell) [inline src/shared/translations/nl.json:1802]: should this translation be empty?

File: src/shared/translations/nl.json [hunk 3]
```diff
@@ -1622,7 +1622,7 @@
   "collection/components/collection-or-bundle-edit___een-bundel-met-redactie-moet-een-kwaliteitscontrole-verantwoordelijke-hebben": "Een bundel met optie 'redactie' moet een verantwoordelijke kwaliteitscontrole hebben.",
   "collection/components/collection-or-bundle-edit___een-collectie-met-redactie-moet-een-kwaliteitscontrole-verantwoordelijke-hebben": "Een collectie met optie 'redactie' moet een verantwoordelijke kwaliteitscontrole hebben.",
   "collection/components/collection-or-bundle-edit___er-ging-iets-mis-bij-het-toevoegen-van-de-collectie": "Er ging iets mis bij het toevoegen van de collectie.",
-  "collection/components/collection-or-bundle-edit___er-ging-iets-mis-bij-het-toevoegen-van-het-item": "Er ging iets mis bij het toevoegen van het fragment of de collectie.",
+  "collection/components/collection-or-bundle-edit___er-ging-iets-mis-bij-het-toevoegen-van-het-item": "Er ging iets mis bij het toevoegen van het fragment .",
```
Comment (bertyhell) [inline src/shared/translations/nl.json:1625]: space before point:
```suggestion
  "collection/components/collection-or-bundle-edit___er-ging-iets-mis-bij-het-toevoegen-van-het-item": "Er ging iets mis bij het toevoegen van het fragment.",
```

**Review by bertyhell — CHANGES_REQUESTED** [review]

File: src/collection/components/CollectionOrBundleOverview.tsx (hunk 1 above)
Comment (ian-emsens-shd) [reply src/collection/components/CollectionOrBundleOverview.tsx:408]: mapping bloat, onnodige `switch/case`-statements, string-matching (`createQuickLane === ceateQuickLane`), ...

**Pull request merged**